Soyeon/MyProfile/AccountStatusView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  AccountStatusView.swift
+//  Soyeon
+//
+//  Created by junyeong-cho on 2021/09/17.
+//  Copyright © 2021 ludus. All rights reserved.
+//
+
+import UIKit
+
+final class AccountStatusView: XibView {
+    
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var coupleDateLabel: UILabel!
+    
+    private var formatter = defaultFormatter
+    
+    override final class var layerClass: AnyClass {
+        return CAGradientLayer.self
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupLayout()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        setupLayout()
+    }
+    
+    private func setupLayout() {
+        (layer as? CAGradientLayer)?.colors = [
+            UIColor(red: 111/255, green: 121/255, blue: 231/255, alpha: 1.0).cgColor,
+            UIColor(red: 136/255, green: 111/255, blue: 212/255, alpha: 1.0).cgColor
```

Comment:
RGB 색상 생성 구문이 반복 되어요 (#105 여기 PR 부분에도요!) 
간단 이니셜라이저로 만들어서 사용하는건 어떨까요? 

```suggestion

UIColor(r: 111, g: 121, b: 231).cgColor
UIColor(r: 136, g: 111, b: 212).cgColor

extension UIColor {
    convenience init(r: Int, g: Int, b: Int) {
        self.init(red: CGFloat(r) / 255.0,
        green: CGFloat(g) / 255.0,
        blue: CGFloat(b) / 255.0,
        alpha: 1.0)
    }
}
```

Reply:
좋은 의견 감사합니다 수정해보도록 할게요!